### PR TITLE
update intervale documetation with covid reopening rules

### PR DESCRIPTION
**Pull request by PhiloEpisteme**
This PR updates the Intervale documentation with COVID-19 rules. The rules themselves come from coordinating with the Karina, the president of MITOC, and the Office of General Counsel and SOLE.

### CODE REVIEW

**Comment by PhiloEpisteme** [comment]
@fpagnoux I'll update the PR with the correct URLs once I get them from Karina. I'll also hold off on merging until Karina gives me the thumbs up on the content from a rules perspective.

File: _pages/rentals/intervale-manual.md
Comment (fpagnoux) [inline _pages/rentals/intervale-manual.md:24]:
```suggestion
- **Occupancy:** The cabin and yurt are indoor spaces and may be reserved separately. Only 1 group of up to 10 people may reserve the cabin or yurt for any single night. 
```

File: _pages/rentals/intervale-manual.md
Comment (fpagnoux) [inline _pages/rentals/intervale-manual.md:14]:
```suggestion
### <span style="color:red;">Outdoor/Camping Only</span>
```

**Review by fpagnoux — APPROVED** [review]
Looks good, good to merge once the URL is fixed and the Karina approves

**Comment by PhiloEpisteme** [comment]
@fpagnoux can you, @DavidCain or @mprat would one of you mind merging this PR. Karina has given her thumbs up.

**Pull request merged**
